crds/submit/monitor.py

```diff
@@ -43,14 +49,14 @@ def add_args(self):
     def main(self):
         """Main control flow of submission directory and request manifest creation."""
         log.divider("monitoring server on " + repr(self.args.key), char="=")
-        exit_flag = False
-        while not exit_flag:
+        exit_status = None
+        while exit_status is None:
             for message in self._poll_status():
                 handler = getattr(self, "handle_" + message.type, self.handle_unknown)
-                exit_flag = handler(message)
+                exit_status = handler(message)
             time.sleep(self.args.poll_delay)
         log.divider("monitoring server done", char="=")
-        return exit_flag
```

Comment:
I think previously this had been returning the URL and calling sys.exit(...) with it?  I don't know if anyone will actually be using the exit status of the (standalone) monitor script, but it seemed worth fixing.